app/views/helpers/TimelineEvents.scala

```diff
@@ -92,12 +93,26 @@ class TimelineEvents @Inject() (
     }
   }
 
-  private def amendmentRejectedAsLatest(submission: Submission): Boolean =
-    submission.latestAction.exists { action =>
-      val isAmendmentRejected = action.requestType == AmendmentRequest
-      val hasLatestErrorNotification = action.notifications.headOption.exists(_.headOption.exists(_.enhancedStatus == ERRORS))
-      submission.blockAmendments == false && isAmendmentRejected && hasLatestErrorNotification
-    }
+  sealed trait AmendmentAsLatest
```

Comment:
I think this would make more sense as `AmendmentResult` or `AmendmentOutcome`

Reply:
And then we can avoid the `maybe` prefix in variable names, because `None` on an `AmendmentOutcome` makes sense (to me at least!)

Reply:
What about `AmendmentAsLatestOutcome`? Because we are checking here if the `AmendmentRequest` action/notification is actually the latest one.

Reply:
Sorry to correct... but it's the possible outcome of an amendment action and notification **as latest**.
If not latest `getAmendmentEventIfLatest()` will return `None`.

Reply:
imo, just name it as `AmendmentOutcome` could confuse in future who could add other amendment status even when the action/notification is not the latest.

Reply:
So harking back to your earlier comment Lucio - would `LatestAmendmentOutcome` work?